AmplifyPlugins/API/Tests/APIHostApp/AWSAPIPluginLazyLoadTests/LL11/GraphQLLazyLoadPostComment8Tests.swift

```diff
@@ -61,7 +61,7 @@ final class GraphQLLazyLoadPostComment8Tests: GraphQLLazyLoadBaseTest {
             return
         }
         assertList(comments, state: .isNotLoaded(associatedIdentifiers: [post.postId, post.title],
-                                                 associatedField: "postId"))
+                                                 associatedFields: ["postId", "postTitle"]))
```

Comment:
Updated codegenerated schema files specify the targetNames, which are passed to the lazy list when instantiating it, thus a not loaded list will have the necessary field names of the post, that exist on the Comment, to query for the Comments by post identifiers.